### PR TITLE
implemented solution

### PR DESCRIPTION
**Pull request by willwildred**
- [DEMO LINK](https://willwildred.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://willwildred.github.io/layout_product-cards/report/html_report/)

- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

**Comment by willwildred** [comment]
Watch out! I have updated my deployment, but there's no actual changes!
You can check out my last commit with button fixes, so everything there should be just fine.
It's just a github...

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    font-weight: 400;
+    color: $price-text-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+    color: #000;
+  }
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:60]: Use scss nesting, fix it everywhere
```suggestion
  &__price {
    margin-bottom: 16px;
    display: flex;
    justify-content: space-between;
    align-items: center;
    
    &-text {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```

File: src/styles/utils/variables.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+$card-border-color: #f3f3f3;
+$main-text-color: #060b35;
+$product-code-text-color: #616070;
+$price-text-color: #616070;
+$service-info-text-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/styles/utils/variables.scss:5]: This value is default, so you can just remove it
```suggestion
```

File: src/styles/utils/variables.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+$card-border-color: #f3f3f3;
+$main-text-color: #060b35;
+$product-code-text-color: #616070;
+$price-text-color: #616070;
```
Comment (volodymyr-soltys97) [inline src/styles/utils/variables.scss:4]: Don't need to duplicate variables with the same values
```suggestion
$product-code-text-color: #616070;
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, you need to make a deploy again after fix text position in the button 
image

This part of the task is incomplete, you need to use styles block from the previous task 
image
image

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Almost done.
Try to fix tests.
Check vertical sizes, line heght for text with design.
It should help to fix some mismatches with reference.

**Review by vadiimvooo — APPROVED** [review]
Well done

**Pull request closed without merging**